build-system/amp.extern.js

```diff
@@ -445,3 +445,14 @@ AMP.require;
  * @typedef {function(number, boolean):?|function(number):?}
  */
 var TransitionDef;
+
+/**
+ * See src/consent-state.js for more information.
+ * @enum {number}
+ */
+const CONSENT_POLICY_STATE = {
```

Comment:
Given this is now externed can we delete src/consent-state.js (and carry-over documentation)?